src/variables.py

```diff
@@ -28,3 +28,10 @@
 HEALTHCHECK_SERVER_PORT = int(os.getenv('HEALTHCHECK_SERVER_PORT', 9010))
 
 MAX_CYCLE_LIFETIME_IN_SECONDS = int(os.getenv("MAX_CYCLE_LIFETIME_IN_SECONDS", 300))
+
+# Oracle constants
+# REWARDS_HISTORICAL_PERIOD default - a one week
+REWARDS_HISTORICAL_PERIOD_IN_SECONDS = int(604800)
```

Comment:
Lets move it to constants.py

Reply:
Sure, but I didn't have constants.py in my branch =). Of course I will